whatsapp_sdk.gemspec

```diff
@@ -46,7 +46,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency('sorbet')
   spec.add_dependency('sorbet-runtime')
 
-  spec.add_dependency("faraday", "~> 2.3.0")
+  spec.add_dependency("faraday", ['>= 2.3.0', '< 3.0'])
```

Comment:
Could you remove the higher version? The usage of this is minimal so it should be very low risk to have no max version.

Reply:
yeah should be fine now